admin/admin-filters-term.php

```diff
@@ -658,12 +658,18 @@ public function get_inserted_term_language( $lang ) {
 		}
 
 		// Special cases for default categories as the select is disabled.
-		elseif ( ! empty( $_POST['tag_ID'] ) && in_array( intval( get_option( 'default_category' ) ), $this->model->term->get_translations( (int) $_POST['tag_ID'] ), true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
-			$lang = $this->model->term->get_language( (int) $_POST['tag_ID'] ); // phpcs:ignore WordPress.Security.NonceVerification
+		elseif ( ! empty( $_POST['tag_ID'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			$default_term = get_option( 'default_category' );
+			if ( is_int( $default_term ) && in_array( $default_term, $this->model->term->get_translations( (int) $_POST['tag_ID'] ), true ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+				$lang = $this->model->term->get_language( (int) $_POST['tag_ID'] ); // phpcs:ignore WordPress.Security.NonceVerification
+			}
```

Comment:
I'm sure you're aware of this but we're changing the behavior here.
Previously we were entering this statement if we add a `tag_ID` AND if it was in the list of translations. Now it only requires `tag_ID` to not be empty.

Meaning:
Previously: non empty `tag_ID` + not a translation = go to next test about `tax_ID`.
Now: non empty `tag_ID` + not a translation = enter this test and do nothing.

To trigger a bug it would require to have both `tag_ID` and `tax_ID` so I suppose you're expecting this not to happen?

Reply:
I rewrote this differently in https://github.com/polylang/polylang/pull/1278/commits/4ae13f8681c683073e39548fac36964e3f4aff1a